tests/product/test_collect.py

```diff
@@ -202,9 +202,9 @@ def _add_custom_log_location(self, new_log_location):
 
     def _collect_logs_and_unzip(self):
         self.run_prestoadmin('collect logs')
-        self.assert_path_exists(self.cluster.master, OUTPUT_FILENAME_FOR_LOGS)
+        self.assert_path_exists(self.cluster.master, TMP_PRESTO_DEBUG + OUTPUT_FILENAME_FOR_LOGS)
```

Comment:
can we have another constant for `TMP_PRESTO_DEBUG + OUTPUT_FILENAME_FOR_LOGS`? or method?

Reply:
Done.